test/functional/rpc_invalid_address_message.py

```diff
@@ -2,19 +2,17 @@
 # Copyright (c) 2020 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
-"""Test error messages for 'getaddressinfo' and 'validateaddress' RPC commands."""
+"""Test error messages for 'validateaddress' RPC command."""
 
 from test_framework.test_framework import BitcoinTestFramework
 
 from test_framework.util import (
     assert_equal,
-    assert_raises_rpc_error,
 )
 
 BECH32_VALID = 'bcrt1qtmp74ayg7p24uslctssvjm06q5phz4yrxucgnv'
 BECH32_INVALID_BECH32 = 'bcrt1p0xlxvlhemja6c4dqv22uapctqupfhlxm9h8z3k2e72q4k9hcz7vqdmchcc'
 BECH32_INVALID_BECH32M = 'bcrt1qw508d6qejxtdg4y5r3zarvary0c5xw7k35mrzd'
-BECH32_INVALID_VERSION = 'bcrt130xlxvlhemja6c4dqv22uapctqupfhlxm9h8z3k2e72q4k9hcz7vqynjegk'
```

Comment:
Instead of removing this, I think a test should added instead?

Reply:
Sure. Test added.